client/src/utils/parser/robotCodeToSsotParsing/robotCodeToSsotParsing.js

```diff
@@ -159,6 +161,30 @@ const currentLineWithoutOutputVariableName = (
   return currentLine;
 };
 
+const returnMatchingCombination = (rpaTask, allMatchingCombinations) => {
+  if (allMatchingCombinations.length === 0) {
+    customNotification(
+      'Error',
+      `The described task "${rpaTask}" could not be assigned to an application.`
+    );
+    return undefined;
+  }
+  if (allMatchingCombinations.length > 1) {
+    if (rpaTask === 'Open Application' || rpaTask === 'Open Workbook') {
+      let correctExampleText = '';
+      allMatchingCombinations.forEach((singleCombination) => {
+        correctExampleText += `\n${singleCombination.Application}.${rpaTask}`;
+      });
+      customNotification(
+        'Error',
+        `Multiple Applications with task "${rpaTask}" found. Give the full name you want to use like: ${correctExampleText}`
+      );
+      return undefined;
+    }
+  }
+  return allMatchingCombinations[0];
+};
+
 /**
  * @description "preprocesses" the code in a usable data format
  * @param {Array} robotCodeTaskSection robot code w/o empty lines as an array of Strings
```

Comment:
param should be updated to taskAndApplicationCombinations

Reply:
is fixed